### DISCUSSION TITLE
How to handle merge conflicts?

### DISCUSSION THREAD

**Discussion by Gurjista** [dotnet/ResXResourceManager#652] [Q&A] [answered]
Hi im learning how to use the extension for my project, and i want to know how to handle merge conflicts in the resx files.
For testing I created a 2 branches that started with two resources, and on each one i created a different one and tried to merge them, and I got a merge conflict.
I've read in a StackOverflow post that if the extension handles merge problems like these if i checked the configuration "When saving a RESX file, sort the XML data nodes by the key", but i don't think that worked for me.
For clarification I've set the code generator of the resx files to public, so I'm not using the custom one that ResXManager provides.
So what should I do to solve the problem?

![image](https://github.com/user-attachments/assets/cc848f2c-0d6d-4ebc-9122-3ba716379bbe)
Branch 1

![image](https://github.com/user-attachments/assets/27920482-90c8-4410-893d-4f7e7eb80ce0)
Branch 2

![image](https://github.com/user-attachments/assets/2ed1e7a7-1aa0-44ce-860c-5ab9accde245)
Merge conflict

**Comment by tom-englert** [comment, marked as answer]
You do like for every other merge conflict in your code. 

I'm using BeyondCompare as the three-way merge tool, so I simply select "Take left then right".

ResXResourceManager can't help you solving merge conflicts.
